ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDApplicant.java

```diff
@@ -14,4 +14,7 @@
     private String representativeOrganisationEmail;
     private String representativeOrganisationDxAddress;
     private CCDYesNoOption leadApplicantIndicator;
+    private String preferredCourtName;
+    private CCDAddress preferredCourtAddress;
+    private String preferredCourtReason;
```

Comment:
Don't we need these fields in CMC domain as well along with their mappings?

Reply:
There's some partial overlap with DQ (I am not working on the story) - Sanchita had added an AC in those DQ stories so as to these fields would be mapped/populated as part of saving DQ to the DB. I think @KiranVarma02 is on the case